docs/Versioning.md

```diff
@@ -127,6 +127,24 @@ Notes:
 - Values that are new or updated from a previous OpSet version are in **bold**.
 - \*: the operator is deprecated.
 
+### Operator Deprecations
+
+The following deprecated operators should be removed from models, substituted with newer operators, or decomposed:
```

Comment:
I think these are all experimental operators, right? You should clarify that these are experimental ops that have been deprecated. The concept of experimental ops is also deprecated now.

Non-experimental operators will remain in documentation even after deprecation. Example is Scatter op which was deprecated in opset 11